dpkg/config.yml

```diff
@@ -0,0 +1,60 @@
+log-type: Syslog
+keys-file: /var/lib/tox-node/keys
+udp-address: 0.0.0.0:33445
+tcp-addresses:
+  - 0.0.0.0:33445
+tcp-connections-limit: 16384
+lan-discovery: True
+threads: 1
+motd: "Hi from tox-rs v0.0.7-alpha! I'm up {{uptime}}. TCP: incoming {{tcp_packets_in}}, outgoing {{tcp_packets_out}}, UDP: incoming {{udp_packets_in}}, outgoing {{udp_packets_out}}"
```

Comment:
Do we need to specify version here? We have it in special motd field. If we specify it here we will have to update it every new release which is tending to be forgotten.

Reply:
Oh, that's a mistake